android/src/main/java/com/reactnativecommunity/imageeditor/ImageEditorModule.java

```diff
@@ -170,6 +170,7 @@ public boolean accept(File dir, String filename) {
   public void cropImage(
       String uri,
       ReadableMap options,
+      boolean useExternalCache,
```

Comment:
copying my comment over from the other PR: as we're already passing `ReadableMap options` above, let's include this flag in there to avoid changing the function signature (which would be a breaking change

Reply:
I see what you mean,
however it's not really a breaking change to the end user since the argument in the JS frontend of this function is optional:
https://github.com/ExodusMovement/react-native-image-editor/pull/1/files#diff-875ccd207727c76a362129948325fbc8486e0d42b2cc0507ac8de5f7bf1e4d6cR65

The reason I didn't put it in the ReadableMap was because it seems to be settings more related to the image.
Width, height etc.

But of course I can move this option into there as well

Reply:
The options argument is actually named "ImageCropData" in the type definition.
That's why I'm thinking it's not the place where this should go

Reply:
@sebbekarlsson good thought! I'd say it's a poorly named type, and it should really be ImageCropOptions

Reply:
@mvayngrib  Fixed